internal/website/content/howto/runtimevar/_index.md

```diff
@@ -175,6 +175,14 @@ https://godoc.org/gocloud.dev/runtimevar/awsparamstore#OpenVariable
 
 ### etcd {#etcd}
 
+*NOTE*: Support for `etcd` has been temporarily dropped until they sort out
```

Comment:
I'd tone it down a bit.

    Support for `etcd` has been temporarily dropped due to dependency issues. See ...

Reply:
OK, done.